src/ralph/data_center/tests/factories.py

```diff
@@ -115,6 +117,44 @@ class Meta:
         model = DataCenterAsset
 
 
+class DataCenterAssetFullFactory(DataCenterAssetFactory):
+    """
+    Factory for DataCenterAsset and m2m relations
+    """
+    rack = factory.SubFactory(RackFactory)
+
+    # m2m relations
+    # TODO: parent, networks (as terminators), operations, security scans,
+    # clusters, tags
+    eth1 = factory.RelatedFactory(
+        EthernetWithIPAddressFactory,
+        'base_object',
+        ipaddress__is_management=True,
+    )
+    eth2 = factory.RelatedFactory(EthernetFactory, 'base_object')
+    eth3 = factory.RelatedFactory(
+        EthernetWithIPAddressFactory,
+        'base_object',
+        ipaddress__dhcp_expose=True,
+    )
+    licence1 = factory.RelatedFactory(
+        'ralph.licences.tests.factories.BaseObjectLicenceFactory', 'base_object'
+    )
+    licence2 = factory.RelatedFactory(
+        'ralph.licences.tests.factories.BaseObjectLicenceFactory',
+        'base_object',
+        quantity=3
+    )
+    support1 = factory.RelatedFactory(
+        'ralph.supports.tests.factories.BaseObjectsSupportFactory',
+        'baseobject'
+    )
+    support2 = factory.RelatedFactory(
+        'ralph.supports.tests.factories.BaseObjectsSupportFactory',
+        'baseobject'
+    )
```

Comment:
Shouldn't be done this way?
http://factoryboy.readthedocs.io/en/latest/recipes.html#simple-many-to-many-relationship